poker-tournament-visualization/src/app/dev-menu/dev-menu.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="buttonContainer" (click)="toggleMenu()">
+  <div class='menuButton'></div>
+</div>
+<div class="devMenu" *ngIf="showMenu">
+  <div class="options">
+    <h2>Load json</h2>
+    <input type="file" id="fileInput" (change)="loadJsonFile($event)" />
+    <p class="inputTitle">Remote json url:</p>
+    <input name="jsonUrl" [(ngModel)]="jsonUrl" #ctrl="ngModel" type="text" placeholder="http://url..."/>
+    <button (click)="loadJsonUrl()">Get Now</button>
+    <h3>{{activeJSON == "" ? "No active json" : "Active game: "  + activeJSON}}</h3>
```

Comment:
Tom streng er falsie right? så ingen grund til at lave comparison, bare vend ternary'en om
Pls corrrect me if im wrong @fredpetersen 
```suggestion
    <h3>{{activeJSON ? "Active game: "  + activeJSON : "No active json"}}</h3>
```